ddtrace/appsec/_iast/_patch.py

```diff
@@ -170,3 +170,21 @@ def _on_iast_fastapi_patch():
         functools.partial(_patched_fastapi_function, OriginType.HEADER),
     )
     _set_metric_iast_instrumented_source(OriginType.HEADER)
+
+    # Instrumented on _iast_starlette_scope_taint
+    _set_metric_iast_instrumented_source(OriginType.PATH_PARAMETER)
+
+
+def _iast_instrument_starlette_scope(scope):
+    if _is_iast_enabled():
```

Comment:
nit: If we need to check `if _is_iast_enabled():` to import this function to begin with, do we need this check inside?

Reply:
You're right, I've been overly careful! :sweat_smile: